Commencement.Mvc/Controllers/AdminController.cs

```diff
@@ -340,10 +340,16 @@ public ActionResult MoveMajor()
         }
 
         [HttpPost]
-        public ActionResult MoveMajor(string majorCode, int ceremonyId)
+        public ActionResult MoveMajor(string majorCode, int? ceremonyId)
         {
+            if (string.IsNullOrWhiteSpace(majorCode) || ceremonyId.HasValue == false)
+            {
+                Message = "Required values not entered";
+                var viewModel = MoveMajorViewModel.Create(Repository, CurrentUser, _ceremonyService);
+                return View(viewModel);
+            }
             var major = _majorRepository.GetNullableById(majorCode);
-            var ceremony = Repository.OfType<Ceremony>().GetNullableById(ceremonyId);
+            var ceremony = Repository.OfType<Ceremony>().GetNullableById(ceremonyId.Value);

```

Comment:
@srkirkland Up a bit, without that, it just got an exception instead of a nice message.